Packs/Devo/Integrations/Devo_v2/Devo_v2.py

```diff
@@ -272,10 +273,12 @@ def fetch_incidents():
         alert_filters = check_type(FETCH_INCIDENTS_FILTER, dict)
 
         if alert_filters['type'] == 'AND':
-            filter_string = ' , '.join([f'{filt["key"]} {filt["operator"]} "{urllib.parse.quote(filt["value"])}"'
+            filter_string = ' , '.join([f'{filt["key"]} {filt["operator"]} '
+                                        f'"{str(filt["value"]) if URLLIB_PARSE else urllib.parse.quote(filt["value"])}"'
                                        for filt in alert_filters['filters']])
         elif alert_filters['type'] == 'OR':
-            filter_string = ' or '.join([f'{filt["key"]} {filt["operator"]} "{urllib.parse.quote(filt["value"])}"'
+            filter_string = ' or '.join([f'{filt["key"]} {filt["operator"]} '
+                                        f'"{str(filt["value"]) if URLLIB_PARSE else urllib.parse.quote(filt["value"])}"'
```

Comment:
let's move to a seperate func so we can test this, as the integrations does not use `requests`.